spec/controllers/workflows_controller_spec.rb

```diff
@@ -31,7 +31,7 @@
   end
 
   describe 'deprecated PUT create' do
-    let(:druid) { 'druid:abc123' }
+    let(:druid) { 'druid:ab123ab1234' }
```

Comment:
note: had to fix dummy values in the tests so it wouldn't trigger the new validation

Reply:
:sunglasses: